### PR TITLE
Added polish translations for Firewall and SQL extensions

### PR DESCRIPTION
**Pull request by mjamro**
Added polish translations for Firewall and SQL extensions

### CODE REVIEW

**Comment by rseanhall** [comment]
We've moved to the .NET Foundation, but we're having trouble getting the bot to check older pull requests.  Here's what it would normally say:

Hi @mariuszjamro, I'm your friendly neighborhood .NET Foundation Pull Request Bot (You can call me DNFBOT). Thanks for your contribution!

In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla2.dotnetfoundation.org.

TTYL, DNFBOT;

**Comment by mjamro** [comment]
Done!

File: src/ext/FirewallExtension/wixlib/pl-pl.wxl [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<!--
+  <copyright file="pl-pl.wxl" company="Outercurve Foundation">
+    Copyright (c) 2004, Outercurve Foundation.
+    This software is released under Microsoft Reciprocal License (MS-RL).
+    The license and further copyright text can be found in the file
+    LICENSE.TXT at the root directory of the distribution.
+  </copyright>
+-->
+<WixLocalization Culture="pl-pl" xmlns="http://wixtoolset.org/schemas/v4/wxl">
+    <String Id="msierrFirewallCannotConnect" Overridable="yes">Nie uda�o si� po��czy� z Zapor� systemu Windows.  ([2]   [3]   [4]   [5])</String>
```
Comment (rseanhall) [inline src/ext/FirewallExtension/wixlib/pl-pl.wxl]: These characters look like the ones you fixed in your last commit for the SqlExtension translation.  Can you fix these, too?

File: src/ext/FirewallExtension/wixlib/pl-pl.wxl [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  <copyright file="pl-pl.wxl" company="Outercurve Foundation">
+    Copyright (c) 2004, Outercurve Foundation.
+    This software is released under Microsoft Reciprocal License (MS-RL).
+    The license and further copyright text can be found in the file
+    LICENSE.TXT at the root directory of the distribution.
+  </copyright>
+-->
```
Comment (rseanhall) [inline src/ext/FirewallExtension/wixlib/pl-pl.wxl]: We've updated our copyright, please replace this with the new one.

File: src/ext/SqlExtension/wixlib/pl-pl.wxl [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  <copyright file="pl-pl.wxl" company="Outercurve Foundation">
+    Copyright (c) 2004, Outercurve Foundation.
+    This software is released under Microsoft Reciprocal License (MS-RL).
+    The license and further copyright text can be found in the file
+    LICENSE.TXT at the root directory of the distribution.
+  </copyright>
+-->
```
Comment (rseanhall) [inline src/ext/SqlExtension/wixlib/pl-pl.wxl]: We've updated our copyright, please replace this with the new one.

**Review by rseanhall — COMMENTED** [review]
Thanks, can you also add a history entry as described [here](https://github.com/wixtoolset/wix4/tree/develop/history)?

**Comment by mjamro** [comment]
Fixed the issues you noticed, please check now.

**Pull request closed without merging**